### ISSUE TITLE
Add "E2E" test that revises a sequence

### ISSUE THREAD

**Issue by corneliusroemer** [loculus-project/loculus#3238]
We don't seem to have one, should be simple, would have caught https://github.com/loculus-project/loculus/issues/3224

(It's easy to write lots of tests that don't ever catch anything useful. So it's good to write tests when something was missed and that thing is easy to test - as in that case one definitely tests something that's relevant)

1. After a sequence has been released by a user
2. Navigate to the seq details page
3. Click the "revise" button
4. Make a small change
5. Submit
6. Check result

Not all of this might be possible with our current setup, that's not a problem, bug #3224 would have been caught when performing step 3/4.

**Comment by theosanderson** [comment]
Yeah I'm going to try to merge the new integration tests pretty soon to allow this sort of thing

**Comment by theosanderson** [comment]
We now have integration tests:

<img width="831" alt="Image" src="https://github.com/user-attachments/assets/85744969-8e32-49e6-a313-29f086d18c9e" />
